web-app/package.json

```diff
@@ -3,16 +3,21 @@
   "description": "Pub Golf Web App",
   "version": "0.0.1",
   "scripts": {
-    "tailwind": "npx tailwind build static/tailwind.config.css -o static/tailwind.css",
-    "protoc": "protoc -I=../proto pubgolf.proto --js_out=import_style=commonjs:./src/proto --grpc-web_out=import_style=commonjs,mode=grpcwebtext:./src/proto",
-    "dev": "sapper dev",
-    "build": "sapper build",
-    "export": "sapper export",
-    "start": "node __sapper__/build",
```

Comment:
It makes sense to alphabetize the rest of this, but I kind of feel like the high-level tasks (`dev`, `build`, `export`, `start`, and `test`) and the lifecycle tasks (currently just `postinstall`) should be  distinct...

What do you think about this order?
```
postinstall

...other stuff (alphabetized)

build
dev
export
start
test
```

Reply:
Sounds good. I with that JSON allowed comments so we could delimit the sections, but YOLO.